release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,74 @@ vf:fulfills     a owl:ObjectProperty ;
 #        rdfs:range      xsd:boolean ;
 #        rdfs:comment    "The commitment or process is complete or not." .
 
-# isSubstitutable "Defines if any resources of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc."
+vf:isSubstitutable a    owl:DatatypeProperty ;
```

Comment:
This seems like the only reason for `vf:ResourceType`. I think we have an issue where we discuss substitutable https://github.com/valueflows/resource/issues/25

Does this stay specific to specific inventory or any would always stay the same for resource using the same taxonomy reference?

Reply:
It is  helpful for planning from a recipe to know if the resource type is substitutable, because if it is not (like say documents), then you can't just find another one in inventory and use that one.  Every resource of that type will "inherit" that, it is a property of the type.